docs/getting-started.md

```diff
@@ -0,0 +1,37 @@
+# Getting started
+
+fcct is a tool that will consume a Fedora CoreOS Config and produce a JSON file that can be given to a Fedora CoreOS machine when it first boots to set the machine up. Using this config, a machine can be told to create users, create filesystems, set up the network, install systemd units, and more.
+
+Fedora CoreOS Configs are YAML files conforming to fcct's schema. For more information on the schema, take a look at [doc/configuration-v1_0.md][1].
+
+As a simple example, let's use ct to set the authorized ssh key for the core user on a Fedora CoreOS machine.
+
+```yaml
+variant: fcos
+version: 1.0.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - ssh-rsa AAAAB3NzaC1yc...
+```
+
+In this above file, you'll want to set the `ssh-rsa AAAAB3NzaC1yc...` line to be your ssh public key (which is probably the contents of `~/.ssh/id_rsa.pub`, if you're on Linux).
+
+If we take this file and give it to fcct:
+
+```
+$ ./bin/fcct --in-file example.yaml
+
+{"ignition":{"config":{"replace":{"source":null,"verification":{}}},"security":{"tls":{}},"timeouts":{},"version":"3.0.0"},"passwd":{"users":[{"name":"core","sshAuthorizedKeys":["ssh-rsa ssh-rsa AAAAB3NzaC1yc..."]}]},"storage":{},"systemd":{}}
+```
+
+We can see that it produces a JSON file. This file isn't intended to be human-friendly, and will definitely be a pain to read/edit (especially if you have multi-line things like systemd units). Luckily, you shouldn't have to care about this file! Just provide it to a booting Fedora CoreOS machine and Ignition, the utility inside of Fedora CoreOS that receives this file, will know what to do with it.
+
+The method by which this file is provided to a Fedora CoreOS machine depends on the environment in which the machine is running. For instructions on a given provider, head over to the [list of supported platforms for Ignition][2].
+
+To see some examples for what else ct can do, head over to the [examples][3].
+
+[1]: configuration-v1_0.md
```

Comment:
Minor: use named anchors over numbered ones for ease of maintenance later on when potentially referencing the same section